runtime/vm/oa64/stackswap.m4

```diff
@@ -0,0 +1,61 @@
+dnl Copyright (c) 2017, 2017 IBM Corp. and others
```

Comment:
If these files are exactly the same as the xa64 versions, can we just use those to avoid having a second copy? I suppose we could rename xa64 to xa64-oa64 or something like that. @gacholio opinions?

Reply:
The other files are common with the differences only in the helpers, so I expect these could also be.

Reply:
That was the approach I was planning to take once I had a version that compiled on OS X.